packages/ppx_tools_versioned-windows.5.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "pierre.boutillier@laposte.net"
+authors: [
+  "Frédéric Bour <frederic.bour@lakaban.net>"
+]
+homepage: "https://github.com/let-def/ppx_tools_versioned"
+bug-reports: "https://github.com/let-def/ppx_tools_versioned/issues"
+license: "MIT"
+tags: "syntax"
+dev-repo: "git://github.com/let-def/ppx_tools_versioned.git"
+build: ["env" "OCAMLFIND_TOOLCHAIN=windows" make "all"]
+install: ["env" "OCAMLFIND_TOOLCHAIN=windows" make "install"]
+remove:
+  ["env" "OCAMLFIND_TOOLCHAIN=windows" "ocamlfind" "remove" "ppx_tools_versioned"]
+depends: [
+  "ocaml-windows"
+  "ocamlfind" {>= "1.5.0"}
```

Comment:
Make it a `{build}` dependency here so the module isn't rebuilt when `ocamlfind` is updated..